From dd0723d5202a102db2b46fd63640c7fa2c93a7f8 Mon Sep 17 00:00:00 2001
From: Cody Peter Mello <cody.mello@joyent.com>
Date: Fri, 3 Nov 2017 15:33:43 +0000
Subject: [PATCH] TOOLS-1677 cr.joyent.us needs to work around Gerrit issue
 1451 TOOLS-1915 Update Gerrit to link to jira.joyent.us Reviewed by: Trent
 Mick <trent.mick@joyent.com> Approved by: Trent Mick <trent.mick@joyent.com>

---
 images/appserver/etc/gerrit.config.base | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/images/appserver/etc/gerrit.config.base b/images/appserver/etc/gerrit.config.base
index 79b5d2f..5482674 100644
--- a/images/appserver/etc/gerrit.config.base
+++ b/images/appserver/etc/gerrit.config.base
@@ -67,11 +67,11 @@
 # trackers.
 #
 [commentlink "devjira"]
-	match = ([A-Z0-9]+-[0-9]+)
-	link = https://devhub.joyent.com/jira/browse/$1
+	match = "(^|\\s|[:.,\\(\\[\\{])([A-Z]+-[0-9]+)\\b"
+	html = "$1<a href=\"https://jira.joyent.us/browse/$2\">$2</a>"
 [commentlink "github"]
-	match = "([a-zA-Z0-9]+)/([-_a-zA-Z0-9]+)#([0-9]+)"
-	link = https://github.com/$1/$2/issues/$3
+	match = "(^|\\s|[:.,\\(\\[\\{])([a-zA-Z0-9]+)/([-_a-zA-Z0-9]+)#([0-9]+)\\b"
+	html = "$1<a href=\"https://github.com/$2/$3/issues/$4\">$2/$3#$4</a>"
 
 #
 # Clarify the purpose of some of the buttons on the Change screen.
-- 
2.21.0

